### ISSUE TITLE
Typos in alt for about section comic strip

### ISSUE THREAD

**Issue by microbit-robert** [microbit-foundation/python-editor-next#650]
In the alt attribute for the comic strip in the read more section of the about dialog there are a couple of typos.

- `severs` should be `servers`
- `the micro:bit` should be `your micro:bit`
